src/reynir_correct/checker.py

```diff
@@ -275,6 +275,8 @@ def annotate(sent: _Sentence) -> List[Annotation]:
                             code=t.error_code,
                             text=t.error_description,
                             detail=t.error_detail,
+                            original=t.error_original if hasattr(t, "original") else "",
```

Comment:
Bug here: Should probably be `t.getattr("original", None)`  (or `t.getattr("error_original", None)`?)

Reply:
Fixed